projects/elife.yaml

```diff
@@ -363,3 +371,48 @@ search:
         ports:
             - 22
             - 80
+
+bus:
+    default-branch: null
+    domain: null
+    int_domain: null
+    aws:
+        ec2: false
+        sns:
+            # bus-articles-prod, bus-articles-end2end, etc.
+            - bus-articles--{cluster}
```

Comment:
Choose a name and I will change it, no preference on how it's named as long as it is consistent. The format is that of Python's `string.format` method.